tests/common.sh

```diff
@@ -47,22 +47,31 @@ initialize_test() {
 #
 finalize_test() {
   local status="$1"
+  shift 1
+  local message=$@
 
   local rc=1
   if [ $status -eq 0 ]; then
       pass
       rc=0
   elif [ $status -eq -1 ]; then
-      skip
-      rc=0
+      skip $message
```

Comment:
`${message}`

Reply:
I just tested doing this, and there is no output from the `fail` function by doing this.